design/workitems.go

```diff
@@ -96,6 +96,7 @@ var _ = a.Resource("workitem", func() {
 			a.Param("page[offset]", d.String, "Paging start position")
 			a.Param("page[limit]", d.Integer, "Paging size")
 			a.Param("filter[assignee]", d.String, "Work Items assigned to the given user")
+			a.Param("filter[iteration]", d.String, "Work Items which are in the given iteration")
```

Comment:
@kwk  if I understand your comment correctly, I will change it to - `IterationID to filter Work Items`.
Right ?